data/items.js

```diff
@@ -378,7 +378,7 @@ let BattleItems = {
 		num: 851,
 		gen: 7,
 		isPokeball: true,
-		desc: "A special Poke Ball designed to catch Ultra Beasts.",
+		desc: "A Poké Ball that has a low success rate for catching a Pokémon.",
```

Comment:
Descriptions are supposed to be clear and useful, you're not supposed to just copy the in-game description.

Reply:
This should probably just be changed back.

Reply:
You’re also not supposed to use accented e because smogon standard is to not use them

Reply:
Do we need to follow Smogon standard?